core/ops_builtin.rs

```diff
@@ -81,3 +91,62 @@ pub fn op_print(
   }
   Ok(())
 }
+
+// Ops related to streaming WebAssembly
+mod wasm {
```

Comment:
We don't use this scoping anywhere else in the project (other than tests). Please remove the extra scope.